### ISSUE TITLE
Running on windows : Problem in generated relativePath in StoryLoader.js

### ISSUE THREAD

**Issue by ayunawani** [elderfo/react-native-storybook-loader#28]
Hi,
While running on windows,  relativePath generated in StoryLoader.js is with **backslash** rather than **forwardslash**. 
Due to this i am getting red screen error while running my react-native app.

```
// template for doT (https://github.com/olado/doT)

function loadStories() {
  
  require('./..\TestComponent\storybook\index.story.js');  // error in this line
  
}

module.exports = {
  loadStories,
};
```

**Comment by ayunawani** [comment]
I have fix this issue in my forked repository . I you want i can send you the pull request.

**Comment by elderfo** [comment]
Go ahead and send a pull request

**Comment by ayunawani** [comment]
Hi @elderfo ,
Just wanted to know that when will you release new version with this pull request ?
As i have dependency on yours project and waiting for this windows fix.

**Comment by elderfo** [comment]
Fixed in version: 1.3.2

**Comment by ayunawani** [comment]
Thanks @elderfo for quick release  👍

**Comment by elderfo** [comment]
A lot of code was refactored to wrap tests around the path resolution. This refactored code was released in [v1.4.2](https://github.com/elderfo/react-native-storybook-loader/releases/tag/v1.4.2)
